types/b2a/index.d.ts

```diff
@@ -1,19 +1,32 @@
-// Type definitions for b2a 1.0
+// Type definitions for b2a 1.1
 // Project: https://github.com/kaelzhang/b2a#readme
 // Definitions by: PatPL <https://github.com/PatPL>
+//                 Merlinio <https://github.com/Merlinio>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /**
  * Converts base64 string back into original text
  */
-declare function atob(base64: string): string;
+ declare function atob(base64: string): string;
```

Comment:
Not a fan of adding that 1 space indentation (auto-formatter issue?) but I don't really mind as long as it stays consistent.

Reply:
You are right. I didn't noticed...

_It's kinda the day of blindness today because it's not the first thing I overlooked._